File: docs/UprimerPay/acquire_sdk_python_v1.0/AcquireSdk/request/tokenRequest.py
import sys
sys.path.append('../') 
from core.request import Request

# 
#   获取token
#   文档地址：https://uprimer.net/open/docs/specification#/jsgf_tksq * 
#  
class TokenRequest(Request):
    def __init__(self):
        super().__init__()

if __name__ == '__main__': 
    request = TokenRequest()
    token = request.getToken()
    print()
    print("token:  ")
    print()
    print(token)

File: docs/UprimerPay/acquire_sdk_php_v1.0/AcquireSdk/request/TokenRequest.php
<?php
namespace AcquireRequest;

require_once dirname(__FILE__)."/../loader.php";

use AcquireSdk\core\Request;

/**
 * 获取token
 * 文档地址：https://uprimer.net/open/docs/specification#/jsgf_tksq * 
 */
$request = new Request();

$token = $request->getToken();
var_dump($token);